Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/VisualUtils.cs

```diff
@@ -14,21 +14,26 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
     internal class VisualUtils
     {
 
-        internal static void HandleIgnoreCollider(Collider handlesIgnoreCollider, List<Transform> handles)
+        internal static void HandleIgnoreCollider(Collider handlesIgnoreCollider, List<Transform> handles, bool ignore = true)
         {
             if (handlesIgnoreCollider != null)
             {
                 foreach (Transform handle in handles)
                 {
                     Collider[] colliders = handle.gameObject.GetComponents<Collider>();
                     foreach (Collider collider in colliders)
-                    {
-                        UnityEngine.Physics.IgnoreCollision(collider, handlesIgnoreCollider);
+                    { 
```

Comment:
nit: extra whitespace

```suggestion
                    {
```